ui/src/SampleUpload.js

```diff
@@ -264,6 +264,8 @@ class SampleUpload extends Component {
           onProcessJob={this.onProcessJob}
           onCancelJob={this.onCancelJob}
           authorisedActivities={this.props.authorisedActivities}
+          loadJobType={"SAMPLE"}
```

Comment:
I think you can safely leave job type out of it and just check the permission... we can clearly see that only `SAMPLE` job types would ever be supplied to this component